src/mobile/awdl.go

```diff
@@ -1,5 +1,6 @@
-package yggdrasil
+package mobile
 
+/*
```

Comment:
Reminder to uncomment and fix this, or else just give up on it entirely if apple isn't going to add support for what we need.